components/resources/library/src/skikoMain/kotlin/org/jetbrains/compose/resources/ImageResources.skiko.kt

```diff
@@ -22,7 +28,10 @@ internal actual fun ByteArray.toImageBitmap(resourceDensity: Int, targetDensity:
         val dstRect = Rect.Companion.makeWH(targetW, targetH)
 
         targetImage = Surface.makeRasterN32Premul(targetW.toInt(), targetH.toInt()).run {
-            val paint = Paint().apply { isAntiAlias = true }
+            val paint = Paint().apply {
+                isAntiAlias = true
+                imageFilter = ImageFilter.makeImage(image, srcRect, dstRect, SamplingMode.MITCHELL)
```

Comment:
In `ImageFilter.makeImage` I am not sure what `srcRect, dstRect` mean, maybe we just pass it to `drawImageRect`? It seems on a lower level

Reply:
Probably, I understood, it will draw the image on any draw operation. So, `drawImageRect` will draw 2 images. Let's just use `drawImageRect`